has-selector.js

```diff
@@ -3,65 +3,81 @@
 (function () {
     "use strict";
 
-    var basis = Date.now(); // Base number used to create unique ids for each alias
-    var outer = [];         // Each entry is a CSS selector corresponding to the outer element
-    var inner = [];         // Each entry is a CSS selector corresponding to the inner element
-    var alias = [];         // Each entry is a unique class name
+    var unique = Date.now(); // Base number used to create unique ids for each alias
+    var outer = [];          // Each entry is a CSS selector corresponding to the outer element
+    var inner = [];          // Each entry is a CSS selector corresponding to the inner element
+    var alias = [];          // Each entry is a unique class name
 
-    addEventListener("mouseenter", toggleAliases);
-    addEventListener("mousedown", toggleAliases);
-    addEventListener("mouseout", toggleAliases);
-    addEventListener("focusin", toggleAliases);
-    addEventListener("change", toggleAliases);
-    addEventListener("click", toggleAliases);
-    addEventListener("input", toggleAliases);
+    parseInlineStylesheets();
+    loadExternalStylesheets();
     addEventListener("blur", toggleAliases);
-    addEventListener("load", toggleAliases);
-    loadStylesheets();
+    addEventListener("input", toggleAliases);
+    addEventListener("click", toggleAliases);
+    addEventListener("change", toggleAliases);
+    addEventListener("focusin", toggleAliases);
+    addEventListener("mouseout", toggleAliases);
+    addEventListener("mousedown", toggleAliases);
+    addEventListener("DOMContentLoaded", toggleAliases);
```

Comment:
Looks like you dropped some listeners, what is the rationale for that? Looks like mouseenter and load were removed.